docs/src/rsh/compute/index.md

```diff
@@ -1899,10 +1899,35 @@ The default value of this argument is a no-op.
 
 The `{!rsh} absoluteTime` and `{!rsh} absoluteSecs` are equivalent to `{!rsh} Left` and `{!rsh} Right` variant tags.
 
+```reach
+load: /examples/nft-auction-api/index.rsh
+range: 48 - 52
+```
+
```

Comment:
I think this looks bad on the page because it seems like something is broken since the code samples are adjacent